action.yml

```diff
@@ -52,6 +52,9 @@ inputs:
     description: "Do not perform tagging, just calculate next version and changelog, then exit."
     required: false
     default: "false"
+  repo_path:
+    description: "Path relative to $GITHUB_WORKSPACE that contains the cloned repository."
```

Comment:
Could you also update the doc in the README (with some info as to when you'd want to do that)?